README.md

```diff
@@ -113,17 +131,16 @@ Krausening uses both the `maven-release-plugin` and the `nexus-staging-maven-plu
 </settings>
 ```
 
-3.) Install `gpg` and distribute your key pair - see [here](http://central.sonatype.org/pages/working-with-pgp-signatures.html). OS X users may need to execute:
+3.) Install `gpg` and distribute your key pair -
+see [here](http://central.sonatype.org/pages/working-with-pgp-signatures.html). OS X users may need to execute:
 
-```
-#!bash
+```Shell
 export GPG_TTY=`tty`;
 ```
 
-4.) Execute `mvn release:clean release:prepare`, answer the prompts for the versions and tags, and perform `mvn release:perform`
+4.) Execute `mvn release:clean release:prepare`, answer the prompts for the versions and tags, and
+perform `mvn release:perform`
 
 ## Licensing
-Krausening is available under the [MIT License](http://opensource.org/licenses/mit-license.php).
 
-## Session Beer
-Prime would like to thank [Counterpointe Solutions](http://cpointe-inc.com/) for providing continuous integration and static code analysis services for Prime.
+Krausening is available under the [MIT License](http://opensource.org/licenses/mit-license.php).
```

Comment:
S: Should be license info for Prime, not Krausening.

Reply:
Good catch - resolved!